internal/cli/config.go

```diff
@@ -82,6 +86,14 @@ func (opts *configOpts) Run() error {
 				Help:    helpLink,
 			},
 		},
+		{
+			Name: "projectId",
+			Prompt: &survey.Input{
+				Message: "Project ID [optional]:",
+				Help:    "This is the ID or an existing project your API keys have access to, You can leave this blank and specify it on every command with --projectId",
```

Comment:
typo? (or sorry if I was the one who did them)
```suggestion
				Help:    "This is the ID of an existing project your API keys have access to, you can leave this blank and specify it on every command with --projectId",
```